File: gamehub-android/game/src/main/java/com/farsitel/bazaar/game/data/RankItem.java
package com.farsitel.bazaar.game.data;

public class RankItem {
    String nickname, score, award;
    boolean hasFollowingEllipsis, isCurrentUser, isWinner;

    public RankItem(String nickname, String score, String award, boolean hasFollowingEllipsis, boolean isCurrentUser, boolean isWinner){
        this.nickname = nickname;
        this.score = score;
        this.award = award;
        this.hasFollowingEllipsis = hasFollowingEllipsis;
        this.isCurrentUser = isCurrentUser;
        this.isWinner = isWinner;
    }
}

File: gamehub-unity/Assets/Bazaar/GameHub/Scripts/Data/RankItem.cs
using UnityEngine;

namespace Bazaar.GameHub.Data
{
    public class RankItem
    {
        public string nickname;
        public string score;
        public string award;
        public bool hasFollowingEllipsis;
        public bool isCurrentUser;
        public bool isWinner;

        public RankItem(AndroidJavaObject entity)
        {
            this.nickname = entity.Get<string>("nickname");
            this.score = entity.Get<string>("score");
            this.award = entity.Get<string>("award");
            this.hasFollowingEllipsis = entity.Get<bool>("hasFollowingEllipsis");
            this.isCurrentUser = entity.Get<bool>("isCurrentUser");
            this.isWinner = entity.Get<bool>("isWinner");
        }

        override public string ToString() => $"nickname: {nickname}, score: {score}";
    }
}